cppcache/src/TcrMessage.cpp

```diff
@@ -838,6 +840,22 @@ void TcrMessage::processChunk(const uint8_t* bytes, int32_t len,
       }
       break;
     }
+    case PUT_DATA_ERROR: {
```

Comment:
`TcrMessage::` ?  for consistency if nothing else

Reply:
Done.